subsys/bluetooth/audio/csip_set_member.c

```diff
@@ -74,13 +101,65 @@ static bool is_last_client_to_write(const struct bt_csip_set_member_svc_inst *sv
 	}
 }
 
-static void notify_lock_value(const struct bt_csip_set_member_svc_inst *svc_inst,
+void csip_gatt_notify_complete_cb(struct bt_conn *conn, void *user_data)
+{
+	/* Notification done, clear bit and reschedule work */
+	atomic_clear(&notify_rdy);
+	k_work_submit(&deferred_nfy_work);
+}
+
+static int csip_gatt_notify(struct bt_conn *conn,
+			    const struct bt_uuid *uuid,
+			    const struct bt_gatt_attr *attr,
+			    const void *data,
+			    uint16_t len)
```

Comment:
```suggestion
static int csip_gatt_notify_set_lock(struct bt_conn *conn,
			    const struct bt_gatt_attr *attr,
			    const void *data,
			    uint16_t len)
```

Since this is only used for the set lock, we can omit the UUID from the parameters (and actually also the `len`) since these are constant values